ext/README.md

```diff
@@ -0,0 +1,12 @@
+# ext folder for projects using kubernetes playgound
```

Comment:
I'd use the term "directory" instead of folder. https://english.stackexchange.com/questions/113606/difference-between-folder-and-directory

Reply:
done thx